package.json

```diff
@@ -6,6 +6,11 @@
   "bin": {
     "plugin-build-edge-handlers": "src/cli.js"
   },
+  "files": [
+    "src",
+    "!*~",
```

Comment:
why do these exist in the first place? if it's an environment specific thing, maybe that should be in a global `.npmignore` file?

Reply:
Fixed.
Those are temporary files created by Vim (and maybe others?).